docs/howto/object-storage/s3/versioning.md

```diff
@@ -53,22 +108,72 @@ the following commands:
     ```bash
     aws --profile <region> \
       --endpoint-url=https://s3-<region>.{{extra.brand_domain}}:8080 \
-      s3api get-object \ 
+      s3api get-object \
       --bucket <bucket-name> \
-	  --key <object-name> \
-	  --version-id <versionid> \
-	  <local-filename>
+      --key <object-name> \
+      --version-id <versionid> \
+      <local-filename>
     ```
 === "mc"
     ```bash
     mc cp \
-	  --version-id <versionid> \
-	  <region>/<bucket-name>/<object-name> \
-	  <local-filename>
+      --version-id <versionid> \
+      <region>/<bucket-name>/<object-name> \
+      <local-filename>
     ```
 === "s3cmd"
     This functionality is not available with the `s3cmd` command.
 
 When you download an object from a versioned bucket *without*
 specifying a version identifier, your S3 client will download the
 latest version of the object that the bucket contains.
+
+
+## Deleting a versioned object
+
+Like the commands to *create* objects, the commands to *delete* them
+behave differently once object versioning is enabled on a bucket.
+
+The command to delete an object will not delete the object altogether,
+but instead revert it to the prior version. The exception to this rule
```

Comment:
I suggest a slight rephrasing: "The command to delete an object will not delete it altogether but revert it to the prior version"

Reply:
Good suggestion, and we can trim it even further by removing the word "altogether".